README.md

```diff
@@ -21,9 +21,10 @@ The master encryption key can be found in PassPack. Place the key under `config/
 
 ### Development
 #### Running local server
-To start a local puma server:
+To start the local server, use webrick (puma is not configured to run locally):
+
+    rails s webrick
```

Comment:
Please note: `rails s` does not work.

Reply:
Is there are reason puma is not configured to run locally? It was able to before.

Reply:
Yes there is a way to set up the configs for each environment, but this is fine for now.